config/components/manager/controller_manager_config.yaml

```diff
@@ -31,3 +31,4 @@ integrations:
   frameworks:
   - "batch/job"
 # - "kubeflow.org/mpijob"
+# - "ray.io/rayjob"
```

Comment:
Can you mention `ray` in the following as well?

https://github.com/kubernetes-sigs/kueue/blob/74f45f69612a2d36ace91ae455f1e9fdbcd53037/apis/config/v1beta1/configuration_types.go#L109-L115